src/api-service/__app__/onefuzzlib/workers/scalesets.py

```diff
@@ -542,14 +552,11 @@ def resize(self) -> None:
             self.set_shutdown(now=True)
             return
 
-        if size == self.size:
-            self._resize_equal()
-        elif self.size > size:
-            self._resize_grow()
-        else:
-            self._resize_shrink(size - self.size)
```

Comment:
We're letting autoscale do the resizing for us. We already handle the case if additional nodes are added in azure.